src/OpenTK.Windowing.Common/Input/MouseCursor.cs

```diff
@@ -0,0 +1,101 @@
+﻿//
+// MouseCursor.cs
+//
+// Copyright (C) 2018 OpenTK
+//
+// This software may be modified and distributed under the terms
+// of the MIT license. See the LICENSE file for details.
+//
+
+using System;
+
+namespace OpenToolkit.Windowing.Common.Input
+{
+    /// <summary>
+    /// Represents a predefined or custom mouse cursor.
+    /// </summary>
+    public sealed class MouseCursor : WindowIcon
+    {
+        private MouseCursor()
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new <see cref="MouseCursor" /> instance from a
+        /// contiguous array of BGRA pixels.
```

Comment:
MouseCursor uses BGRA but WindowIcon documents to use RGBA:
https://github.com/opentk/opentk/pull/867/files#diff-15124a82c2245dedbe20af0dcc7860a1R17

Reply:
You're right, no idea why that is. Will change the docs to reflect this.

Reply:
Fixed.